manifests/input.yaml

```diff
@@ -2,452 +2,352 @@ format: yamanifest
   hashes:
-    binhash: 6049f0c555b3c33d61bc3891f5a397c5
+    binhash: 9401083e4199439b77254fc490b6597d
     md5: 8525fd107dbea6184db74089f1d55fd9
 work/atmosphere/INPUT/TSI_CMIP6_ESM_v2:
-  fullpath: /g/data/access/payu/access-esm/input/historical/atmosphere/TSI_CMIP6_ESM_v2
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/modern/historical/atmosphere/forcing/resolution_independent/2021.06.22/TSI_CMIP6_ESM_v2
   hashes:
-    binhash: cc14c34512c0c1141735af4d5969a9c9
+    binhash: 32f787c9ffdf8902cf642fe218ff76d5
     md5: 15aa13632aa616dcf079b152a5cbd120
 work/atmosphere/INPUT/biogenic_351sm.N96L38:
-  fullpath: /g/data/access/payu/access-esm/input/pre-industrial/atmosphere/biogenic_351sm.N96L38
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/modern/share/atmosphere/aerosol/global.N96/2020.05.19/biogenic_351sm.N96L38
   hashes:
-    binhash: 8ab4ff40358e274aa21d0240c8af2f71
+    binhash: 35f8585c3290dfc25ed19e7d93f6a1bb
     md5: 55eb75f602a4639fb3c8ff40e36eb661
 work/atmosphere/INPUT/cableCMIP6_LC_1850-2015.nc:
-  fullpath: /g/data/access/payu/access-esm/input/historical/atmosphere/cableCMIP6_LC_1850-2015.nc
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/modern/historical/atmosphere/land/vegetation/global.N96/2024.07.04/cableCMIP6_LC_1850-2015.nc
   hashes:
-    binhash: 306f76d4c25c43b70527c9f030bfdd13
-    md5: 425a2b30b996d40e34acd61d319a6faf
+    binhash: fc03d315746c6a0b254a7632dc28295c
+    md5: 78ddfbdc9d9551cf6c6c39fe734716a4
 work/atmosphere/INPUT/cable_vegfunc_N96.anc:
-  fullpath: /g/data/access/payu/access-esm/input/pre-industrial/atmosphere/cable_vegfunc_N96.anc
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/modern/share/atmosphere/land/vegetation/global.N96/2020.05.19/cable_vegfunc_N96.anc
   hashes:
-    binhash: a91053ef6f75a81581f28d0979e4d526
+    binhash: b590c62f20575a65f1f5ee83ec687304
     md5: 7a8cb16191ad29b081514657893530c6
-work/atmosphere/INPUT/ozone_1849_2015_ESM1.anc:
-  fullpath: /g/data/access/payu/access-esm/input/historical/atmosphere/ozone_1849_2015_ESM1.anc
+work/atmosphere/INPUT/def_soil_params.txt:
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/share/atmosphere/land/biogeophysics/resolution_independent/2020.05.19/def_soil_params.txt
+  hashes:
+    binhash: f6bbe84fb16971f00d2ba34eef542976
+    md5: 93107e8c37e04ccf97471fd2ac8f63eb
+work/atmosphere/INPUT/def_veg_params.txt:
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/share/atmosphere/land/biogeophysics/resolution_independent/2020.05.19/def_veg_params.txt
   hashes:
-    binhash: eecbb2a8383e88211e7ca64ee88bed6a
-    md5: 96cded5d4248c88e4088700778dc8557
-work/atmosphere/INPUT/ozone_1850_ESM1.anc:
-  fullpath: /g/data/access/payu/access-esm/input/pre-industrial/atmosphere/ozone_1850_ESM1.anc
+    binhash: 22ce1f400a832868aefeaa9721ac3a48
+    md5: 1f9a27f0e252bf6c466100573c126789
```

Comment:
`def_veg_params.txt` is a different file but isn't listed as one of the known changed files. 

It's a bit difficult to spot because `ozone_1850_ESM1.anc` has also been deleted, so the diff makes it look like the hashes are associated with a different file.

Reply:
Ah good catch. These files are actually the same in both cases with md5 hash `1f9a27f0e252bf6c466100573c126789`, but because of the removed files github's rendering of the diff has made it look different which is a bit annoying.